docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -131,15 +131,17 @@ $ etcdctl mk /kube-centos/network/config "{ \"Network\": \"172.30.0.0/16\", \"Su
 * Configure flannel to overlay Docker network in /etc/sysconfig/flanneld on the master (also in the nodes as we'll see):
 
 ```shell
+# Flanneld configuration options
+
 # etcd url location.  Point this to the server where etcd runs
-FLANNEL_ETCD="http://centos-master:2379"
+FLANNEL_ETCD_ENDPOINTS="http://centos-master:2379"
 
 # etcd config key.  This is the configuration key that flannel queries
 # For address range assignment
-FLANNEL_ETCD_KEY="/kube-centos/network"
```

Comment:
@thockin fixed in c4fa6d9. It was same example but for `node`. In my case I had both `master` and `node` on one host so I've skipped that step.